templates/experiments/components/experiment_chat.html

```diff
@@ -47,6 +50,9 @@
                   {% include 'experiments/components/experiment_sessions_list_tags.html' with tags=message.tags.all %}
```

Comment:
I think this will show user tags as well as system tags but it should probably only show user tags right?